db/cache_tracker.hh

```diff
@@ -138,6 +140,13 @@ void cache_tracker::insert(rows_entry& entry) noexcept {
     _lru.add(entry);
 }
 
+inline
+void cache_tracker::insert(rows_entry& entry, rows_entry& more_recent) noexcept {
```

Comment:
Usual practice is to put the location first and new data second (as is visible in the _list.insert call below).

Since here the data types are the same, not iterator/T, we could even wrap it in a marker structure so it's clear which one is which.

Reply:
I put the location first.